### PR TITLE
Add method to expose underlying Frame C struct

### PR DESCRIPTION
**Pull request by l0rem1psum**
This PR address issue #43

### CODE REVIEW

File: frame.go [hunk 1]
```diff
@@ -213,3 +213,7 @@ func (f *Frame) Unref() {
 func (f *Frame) MoveRef(src *Frame) {
 	C.av_frame_move_ref(f.c, src.c)
 }
+
+func (f *Frame) Inner() unsafe.Pointer {
```
[the remark this answers is not in the document]
Comment (l0rem1psum) [reply frame.go:217]: I guess it was just a matter of my preference. Now that I think about it, returning `*C.struct_AVFrame` does offer more clarity and type safety. So let's go with that?

[the remark this answers is not in the document]
Comment (l0rem1psum) [reply frame.go:217]: As discussed below, if we were to return `*C.struct_AVFrame`, do you still recommend calling the method `UnsafePointer`? I do not have a particular preference.

[the remark this answers is not in the document]
Comment (asticode) [reply frame.go:217]: I honestly don't know whether returning `*C.struct_AVFrame` is a bad idea, but it feels better so let's go with that 👍

[the remark this answers is not in the document]
Comment (asticode) [reply frame.go:217]: Thing is I want developers to understand that once they use this method they're on their own and this library could present chaotic behavior. Therefore I want to prefix this new method with `Unsafe`, and thought `Pointer` was clear enough to explain what it does. If you have another idea for the suffix, let me know.

[the remark this answers is not in the document]
Comment (l0rem1psum) [reply frame.go:217]: That was a fair consideration. Again, I don't have a particular preference, so let's go with `UnsafePointer`.

**Comment by l0rem1psum** [comment]
@asticode All fixes implemented

**Comment by asticode** [comment]
Thanks for the PR! ❤️

**Pull request merged**